src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -47,6 +47,10 @@ class PropertyAccessor implements PropertyAccessorInterface
     /** @var int Allow magic __call methods */
     public const MAGIC_CALL = ReflectionExtractor::ALLOW_MAGIC_CALL;
 
+    public const BYPASS_PROPERTY_HOOK_NONE = 0;
+    public const BYPASS_PROPERTY_HOOK_READ = 1 << 1;
+    public const BYPASS_PROPERTY_HOOK_WRITE = 1 << 0;
```

Comment:
constants for a bit mask are generally defined in increasing order (so that new cases are added at the end of the list). This reverse order is unusual to me